### PR TITLE
RUN-5093: Include cross origin child windows and iframes in system.getProcessList

### PR DESCRIPTION
**Pull request by tomer-openfin**
#### Description of Change
**Doc update for the modified 'System.getProcessList' call** 

Related  [core PR](https://github.com/HadoukenIO/core/pull/767)

#### Checklist

- [x] PR description included
- [x] `npm test` passes
- [x] relevant documentation was added
- [x] PR title starts with the JIRA ticket [pull request process](https://github.com/openfin/Internal-Wiki/wiki/Pull-Request-Process)
- [x] PR release notes describe the change in a way relevant to app-developers

#### Release Notes
doc update for the modified 'System.getProcessList' call

### CODE REVIEW

File: src/api/window/windowOption.ts [hunk 1]
```diff
@@ -46,3 +46,10 @@ export interface CustomRequestHeaders {
     urlPatterns: Array<string>;
     headers: Array<object>;
 }
+
+export type WindowOptionDiff = {
```
Comment (rdepena) [inline src/api/window/windowOption.ts]: Looks like this has a bit more changes than intended.

Comment (tomer-openfin) [reply src/api/window/windowOption.ts]: Sorry about that... fixed.

**Comment by tomer-openfin** [comment]
Thanks @licui3936 
I've changed the comments. I went with something shorter than the tutorial description to conform with the rest of the comments in system.ts
Let me know if you think that's not a good idea.

**Comment by tomer-openfin** [comment]
fyi - I'm still working on this pr, no need to review it again just yet.

File: src/api/system/system.ts [hunk 2]
```diff
@@ -771,9 +772,9 @@ export default class System extends EmitterBase<SystemEvents> {
     }
 
     /**
-     * Retrieves an array of all of the runtime processes that are currently
-     * running. Each element in the array is an object containing the uuid
-     * and the name of the application to which the process belongs.
+     * Retrieves an array of all the processes that are currently running across all runtimes
+     * Each process has a type, identifying information and diagnostic process information.
+     * Note: chrome debugger tool fires up it's own tab process.
```
Comment (licui3936) [inline src/api/system/system.ts]: typo: it's =>its

File: tutorials/System.getProcessList.md [hunk 3]
```diff
@@ -1,4 +1,6 @@
-Retrieves an array of all of the runtime processes that are currently running
+Retrieves an array of all the processes that are currently running across all runtimes (not including ones in a foreign security realm context).
+<br>Each process has a type (browser, tab or GPU), identifying information (uuid and name of the corresponding parent application) and diagnostic process information.
+<br>Note: chrome debugger tool fires up it's own tab process.
```
Comment (licui3936) [inline tutorials/System.getProcessList.md]: same typo here

**Comment by rdepena** [comment]
Core PR closed.

**Pull request closed without merging**